addons/mail/static/src/core/public_web/discuss_search.scss

```
@@ -14,3 +14,7 @@
         }
     }
 }
+
+.o-mail-DiscussSearch {
+    padding-top: var(--search-padding, #{map-get($spacers, 1)});
```

Following our [coding guidelines](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html?highlight=coding#css-variables), this css variable should be renamed to something like this :
```suggestion
    padding-top: var(--DiscussSearch-padding-top, #{map-get($spacers, 1)});
```
Same for other occurrences in both repo :)